caso 4 - alteração-periodo-dashboard/alteração-periodo-dashboard.md

```diff
@@ -0,0 +1,26 @@
+# Caso de Teste 4: Alteração do Período da Dashboard
+
+| Alterar periodo da Dashboard |          |
+| ---------------- | -------------------- |
+| Módulo           | Verificar alteração do período da dashboard |
+| Funcionalidade   | Este caso de teste verifica se a alteração do período na dashboard atualiza os dados corretamente |
+| Requisito        | O usuário deve estar logado no sistema e na página da dashboard |
+
+| Criticidade | Crítico |
```

Comment:
Se o dashboard parar de funcionar o software não fica inútil, apenas menos útil. Então a critidade pode ser reduzida para algo como "Normal", indicando que este problema não teria tanta prioridade se comparado com um problema no login por exemplo.